package.json

```diff
@@ -5,19 +5,19 @@
   "version": "1.5.20",
   "ungitPluginApiVersion": "0.2.0",
   "scripts": {
-    "start": "node ./bin/ungit",
-    "inspect": "node --inspect ./source/server.js",
+    "start": "exec node ./bin/ungit",
```

Comment:
Why `exec`? This does not work on windows.

Reply:
Ah - did not know that. It's so it will drop the npm executable and therefore clean up a little memory. I'll take it out.